src/aggregator/aggregator/leader_flush_mgr.go

```diff
@@ -181,20 +176,39 @@ func (mgr *leaderFlushManager) Prepare(buckets []*flushBucket) (flushTask, time.
 		}
 	}
 
-	durationSinceLastPersist := time.Duration(nowNanos - mgr.lastPersistAtNanos)
-	if mgr.flushedSincePersist && durationSinceLastPersist >= mgr.flushTimesPersistEvery {
-		mgr.lastPersistAtNanos = nowNanos
-		mgr.flushedSincePersist = false
-		flushTimes := mgr.prepareFlushTimesWithLock(buckets, allShards)
-		mgr.flushTimesManager.StoreAsync(flushTimes)
-	}
-
 	if !shouldFlush {
 		return nil, waitFor
 	}
+
 	return mgr.flushTask, waitFor
 }
 
+func (mgr *leaderFlushManager) UpdateFlushTimes(buckets []*flushBucket) {
```

Comment:
why do we need a new interface method (that the follower doesn't even impl)? can't we just update the flush times at the end of the flush task?